service-03/PUS-3-1.asn1

```diff
@@ -34,19 +34,22 @@ maxTC-3-1-SuperComutatetParameterSetCount INTEGER ::= 536870911
 
 TC-3-1-CreateAHousekeepingParameterReportStructure ::= TC-3-1-CreateAHousekeepingParameterReportStructureGeneric {HousekeepingParameterReportStructure-ID, CollectionInterval, Parameter-ID}
 
-TC-3-1-CreateAHousekeepingParameterReportStructureGeneric {HousekeepingParameterReportStructure-ID-Type, CollectionInterval-Type, Parameter-ID-Type} ::= SEQUENCE
+TC-3-1-CreateAHousekeepingParameterReportStructureGeneric
+  {HousekeepingParameterReportStructure-ID-Type, CollectionInterval-Type, Parameter-ID-Type}
+  ::= SEQUENCE
 {
     housekeepingParameterReportStructure-ID HousekeepingParameterReportStructure-ID-Type,
     collectionInterval CollectionInterval-Type,
 --! n1 UnsignedInteger-Type, moved to acn
     parameter-ID SEQUENCE (SIZE(0 .. maxTC-3-1-Parameter-ID-Count)) OF Parameter-ID-Type,
 --! nfa UnsignedInteger-Type, moved to acn
-    superCommutatedParameterSet SEQUENCE (SIZE(0 .. maxTC-3-1-SuperComutatetParameterSetCount)) OF SuperCommutatedParameterSet
+    superCommutatedParameterSet SEQUENCE (SIZE(0 .. maxTC-3-1-SuperComutatetParameterSetCount)) OF TC-3-1-SuperCommutatedParameterSet
 }
 
-SuperCommutatedParameterSet ::= SuperCommutatedParameterSetGeneric {PUSC-UINT32, Parameter-ID}
+TC-3-1-SuperCommutatedParameterSet ::=
+  TC-3-1-SuperCommutatedParameterSetGeneric {PUSC-UINT32, Parameter-ID}
 
-SuperCommutatedParameterSetGeneric {UnsignedInteger-Type, Parameter-ID-Type} ::= SEQUENCE
+TC-3-1-SuperCommutatedParameterSetGeneric {UnsignedInteger-Type, Parameter-ID-Type} ::= SEQUENCE
```

Comment:
Why wasn't it moved out of this module just like PeriodicGenerationProperties? If we move repeating fields to external files, maybe we should do this to all such structures.

Reply:
k